### ISSUE TITLE
Error while running pipgui in virtualenv

### ISSUE THREAD

**Issue by L04DB4L4NC3R** [GDGVIT/pip-gui#20]
### Runtime environment 
The following is the desktop environment spec:
```
PRETTY_NAME="Kali GNU/Linux Rolling"
NAME="Kali GNU/Linux"
ID=kali
VERSION="2020.1"
VERSION_ID="2020.1"
VERSION_CODENAME="kali-rolling"
ID_LIKE=debian
```
Virtualenv uses `--python=python3`. I used python version `3.7.2`.

### Executed action
The following commands were executed serially
```bash
$ virtualenv --python=python3 venv
$ source venv/bin/activate
$ python setup.py build
$ python setup.py install
$ pipgui
```

### Outcome

![image](https://user-images.githubusercontent.com/30529572/72897125-320e6d00-3d47-11ea-8c07-026658fc4110.png)

### Related issue
Trying [this](https://github.com/Electron-Cash/Electron-Cash/issues/892#issuecomment-450518733)  or [this](https://github.com/Electron-Cash/Electron-Cash/issues/892#issuecomment-450519821) does not work in virtualenv since `--user` is not allowed in the virtual runtime.

`NOTE`: The aforementioned error is pertaining to the virtualenv runtime only, and does not come in a non-virtual environment.

**Comment by Geek-ubaid** [comment]
The error solved in the latest PR #21

**Comment by L04DB4L4NC3R** [comment]
Thank you @Geek-ubaid
